definitions/private-law/json/CaseEvent/Order/CaseEvent-nonprod.json

```diff
@@ -6,7 +6,7 @@
     "Name": "Draft an order",
     "Description": "Draft an order",
     "DisplayOrder": 28,
-    "PreConditionState(s)": "*",
+    "PreConditionState(s)": "AWAITING_SUBMISSION_TO_HMCTS;SUBMITTED_PAID;CASE_ISSUED;JUDICIAL_REVIEW;AWAITING_RESUBMISSION_TO_HMCTS;CASE_WITHDRAWN;ALL_FINAL_ORDERS_ISSUED;PREPARE_FOR_HEARING_CONDUCT_HEARING;DECISION_OUTCOME",
```

Comment:
If we dont need this event in Pending state, do we need this in Draft state?

May be this question is to @shashisk97 ?

Reply:
we checked with service team earlier and response was it should be present

Reply:
thanks both